Aquality.Selenium/src/Aquality.Selenium/Configurations/WebDriverSettings/ChromeSettings.cs

```diff
@@ -30,11 +30,20 @@ public override DriverOptions DriverOptions
                 SetChromePrefs(options);
                 SetCapabilities(options, (name, value) => options.AddAdditionalOption(name, value));
                 SetChromeArguments(options);
+                SetChromeExcludedArguments(options);
                 SetPageLoadStrategy(options);
                 return options;
             }
         }
 
+        private void SetChromeExcludedArguments(ChromeOptions options)
+        {
+            if (SettingsFile.IsValuePresent($"{DriverSettingsPath}.excludedArguments"))
+            {
+                options.AddExcludedArguments(SettingsFile.GetValueList<string>($"{DriverSettingsPath}.excludedArguments"));
```

Comment:
AddExcludedArguments is also supported in EdgeOptions and OperaOptions. Please add similar methods to EdgeSettings and OperaSettings.

Reply:
done